lib/v2/xaut/index.js

```diff
@@ -0,0 +1,71 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    let category = ctx.params.category;
+    const dic_html = { tzgg: 'tzgg.htm', xyyw: 'xyyw.htm', mtbd: 'mtbd1.htm', xshd: 'xshd.htm' };
+    const dic_title = { tzgg: '通知公告', xyyw: '校园要闻', mtbd: '媒体播报', xshd: '学术活动' };
+
+    // 设置默认值
+    if (dic_title[category] === undefined) {
+        category = 'tzgg';
+    }
+
+    const response = await got({
+        method: 'get',
+        url: 'http://www.xaut.edu.cn/index/' + dic_html[category]
+    });
+    const data = response.body;
+    const $ = cheerio.load(data);
+
+    const list = $('.newslist_block a')
+        .slice(0, 10)
+        .map((_, item) => {
+            item = $(item);
+            const temp = item.find('span').text();
+
+            const link = item.attr('href').replace(/^\.\./, 'http://www.xaut.edu.cn');
+            let date = parseDate(temp.slice(-10, -1),'YYYY-MM-DD');
+            let title = temp.slice(0, -10);
+
+            if (category === 'xshd') {
+                date = new Date(parseDate(temp.slice(-11, -1).replace('年', '-').replace('月', '-').replace('日', ''),'YYYY-MM-DD'));
```

Comment:
```suggestion
                date = parseDate(temp.slice(-11, -1).replace('年', '-').replace('月', '-').replace('日', ''),'YYYY-MM-DD');
```
https://github.com/DIYgod/RSSHub/blob/b5be05b1e2bdc18e6fec5ebe137cca9937175ff3/lib/utils/parse-date.js#L32

`parseDate(...)` is already a Date Object.